bigbluebutton-html5/public/locales/en.json

```
@@ -54,6 +54,26 @@
     "app.emojiPicker.skintones.4": "Medium Skin Tone",
     "app.emojiPicker.skintones.5": "Medium-Dark Skin Tone",
     "app.emojiPicker.skintones.6": "Dark Skin Tone",
+    "app.timer.title": "Time",
+    "app.timer.label": "Time",
+    "app.timer.stopwatch.title": "Stopwatch",
+    "app.timer.stopwatch.label": "Stopwatch",
+    "app.timer.timer.title": "Timer",
+    "app.timer.timer.label": "Timer",
```

```suggestion
    "app.timer.title": "Time",
    "app.timer.stopwatch.title": "Stopwatch",
    "app.timer.timer.title": "Timer",
```
Those are not used anymore